app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -30,9 +30,21 @@
       <% end %>
     <% end %>
     <% if history.any? %>
+      <% expand_full_history =
```

Comment:
This is quite a logic for a component - we try and keep the logic inside components fairly straightforward, sticking to templating logic that could be replicated by simpler templating languages like `mustache`, as a longer term goal of GOV.UK Components is to make them available to wider government, and keeping them simpler will make it easier cross-compile to other languages (in theory). 

This tends to keep components simpler and easier to reason about too, which is nice. In this situation the list manipulation/splitting could be avoided by having a two `history` parameters, something like `history` and `history_hidden` (roughly). This would simplify the template and makes the component's programatic interface more explicit, and would support showing more than one piece of history as a default - but it does put more work on to the app calling the component to do that work, which I don't think is too bad. 

What do you think?